src/sentry/snuba/query_subscription_consumer.py

```diff
@@ -116,7 +141,8 @@ def handle_message(
             with metrics.timer(
                 "snuba_query_subscriber.parse_message_value", tags={"dataset": dataset}
             ):
-                contents = parse_message_value(message_value)
+                jsoncodec = JsonCodec(get_schema(logical_topic)["schema"])
```

Comment:
This is still being run once per message since this method is called on every message. It needs to initialized only once when the consumer starts then reused